Sources/SwifterSwift/Foundation/URLRequestExtensions.swift

```diff
@@ -20,6 +20,34 @@ public extension URLRequest {
         self.init(url: url)
     }
 
+    /// cURL command representation of this URL request.
```

Comment:
SwiftLint is warning. You should replace the comment line with the following example :)

`SwifterSwift: cURL command representation of this URL request.`